tests/src/haxepunk/BaseTest.hx

```diff
@@ -0,0 +1,28 @@
+package haxepunk;
+
+import massive.munit.Assert;
+
+class BaseTest
```

Comment:
I wasn't sure how to handle this. Assert has a `throws` function in the git version of munit but that isn't on haxelib yet.